config/default.js

```diff
@@ -274,6 +275,10 @@ module.exports = {
   // See the Discopane config for how this is currently used.
   taarParamsToUse: [],
 
+  // If true, show a link to the beta versions page on the listing page
+  betaVersions: true,
```

Comment:
I made a new 'Feature flags' section in the config (see line 282) so this should move there now.